src/screens/EventSuggestionScreen.tsx

```diff
@@ -0,0 +1,250 @@
+import { NavigationProp } from '@react-navigation/native';
+import React, { useState } from 'react';
+import { useMutation } from 'react-apollo';
+import { Controller, useForm } from 'react-hook-form';
+import { Alert, Keyboard, ScrollView, StyleSheet } from 'react-native';
+import { Divider } from 'react-native-elements';
+
+import {
+  Button,
+  Checkbox,
+  DefaultKeyboardAvoidingView,
+  HeadlineText,
+  HtmlView,
+  Input,
+  RegularText,
+  SafeAreaViewFlex,
+  Wrapper,
+  WrapperVertical
+} from '../components';
+import { colors, consts, normalize, texts } from '../config';
+import { useMatomoTrackScreenView } from '../hooks';
+import { createQuery, QUERY_TYPES } from '../queries';
+
+const { MATOMO_TRACKING, EMAIL_REGEX } = consts;
+
+type TAppUserContent = {
+  address: string;
+  consent: boolean;
+  email: string;
+  eventName: string;
+  message: string;
+  name: string;
+  phone: string;
+};
+
+export const EventSuggestionScreen = ({
+  navigation,
+  route
+}: {
+  navigation: NavigationProp<any>;
+  route: { params: { formIntroText: string } };
+}) => {
+  const [loading, setLoading] = useState(false);
+  const introText = route.params.formIntroText || '';
+
+  const {
+    control,
+    formState: { errors },
+    reset,
+    handleSubmit
+  } = useForm({
+    defaultValues: {
+      address: '',
+      consent: false,
+      email: '',
+      eventName: '',
+      description: '',
+      name: '',
+      phone: ''
+    }
+  });
+
+  useMatomoTrackScreenView(MATOMO_TRACKING.SCREEN_VIEW.FEEDBACK);
```

Comment:
do not track a feedback screen view for this event suggestion screen!
create a new flag to track or remove the tracking please.

Reply:
Yes, you are right. I copied the code from `FeedbackScreen` and forgot about it.